src/index.js

```diff
@@ -1,13 +1,13 @@
 /*
  * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
```

Comment:
Can you leave the license header as is? Applies to all the license headers that were changed.